packages/engine/src/worker/runner/javascript/runner.js

```diff
@@ -200,7 +200,7 @@
       }
     } catch (e) {
       return {
-        pkg_error: String(e.stack), // `.stack` is V8 built-in; TODO: `e.toString()`?
+        pkg_error: e.toString(),
```

Comment:
Does `toString` give a backtrace? There was a TODO because I haven't checked.

Reply:
We don't expect a backtrace here but a JSON string which should be passed as error to the engine. Previously it also was a string, but formatted as stack trace.

Reply:
```suggestion
        pkg_error: JSON.stringify(e.stack),
```
You're right, I tried running it and it didn't have a backtrace. I would change it then though to have one (and also to be more similar to behavior errors).

Reply:
I don't know if I get what you are meaning. Where do you want to generate a backtrace?

Reply:
I just mean the suggestion above with `e.stack`

Reply:
But then we will have again a stacktrace including internal functions

Reply:
Ah I see your point. I think it's ok for package errors, because similarly to engine errors, they shouldn't happen unless there's a bug in our code, whereas behavior errors are caused by bugs in user code.

Reply:
Yep so if you have an error in behaviors, we would get something like this:
```
   0.144500954s ERROR orchestrator::experiment: Got error: "Worker error: JavaScript runner error: Error in package: Error: \"Error: JavaScript error handling is stupid\\n    at Object.behavior [as fn] (eval at load_behaviors (unknown source), <anonymous>:8:1)\\n    at run_task (<anonymous>:222:18)\\n    at Object.run_task (<anonymous>:192:11)\""
```

Which as you can see, shows a bunch of stuff about run_task and load_behaviors and whatever.

But now we have
```
   0.202180992s ERROR orchestrator::experiment: Got error: "Worker error: JavaScript runner error: Error in package: Error: {\"msg\":\"Error: JavaScript error handling is stupid\",\"frames\":[{\"line\":7,\"fn\":\"behavior\"}]}"
Error: experiment had errors
```

Which actually makes sense as it says the error is line 7 in the behavior function.

Reply:
I think the problem @TimDiekmann is when we have other errors within JS code, not just from user behaviors. I'm pretty sure there were other places that used prepareStacktrace no? 

Need to see how this affects errors being thrown from package impls, etc.

Reply:
There would still be an error. It's not as readable as before, but it's okay-ish:
```
CC trace: SyntaxError: missing ) after argument list
```
The file info is missing however. I suggest getting into that when doing the rest of the error handling.

Reply:
Previously was there a stack trace though?

Reply:
Nope, not really. Just checked it.

Reply:
Sorry, I noticed now that the behavior execution package wasn't returning user errors, but simply throwing them, which causes them to be treated the same way as package errors. Maybe I'll make a follow-up PR then.